### ISSUE TITLE
Uncaught Error: Unknown Webpack structure

### ISSUE THREAD

**Issue by marcomow** [pixeldesu/moduleRaid#20]
Hi! I've stumbled upon this error, the webpage is https://www.seb.lv/sites/default/files/calc/public/en/kma.html

The command I run
```
{
    const script = document.createElement('script');
    script.setAttribute('src', 'https://unpkg.com/moduleraid/dist/moduleraid.iife.js');
    script.addEventListener('load', () => window.mR = new ModuleRaid({debug:true}));
    document.head.appendChild(script);
}
```
The error:
> moduleraid.iife.js:1 Uncaught Error: Unknown Webpack structure
    at o.fillModules (moduleraid.iife.js:1:1888)
    at new e (moduleraid.iife.js:1:1001)
    at HTMLScriptElement.<anonymous> (<anonymous>:4:55)

Debug info:
> moduleraid.iife.js:1 [moduleRaid] Pushing moduleRaid.arrayArguments[0] into webpackJsonp failed:
TypeError: window[t.entrypoint].push is not a function
TypeError: window[t.entrypoint].push is not a function
    at https://unpkg.com/moduleraid/dist/moduleraid.iife.js:1:1662
    at Array.forEach (<anonymous>)
    at o.fillModules (https://unpkg.com/moduleraid/dist/moduleraid.iife.js:1:1562)
    at new e (https://unpkg.com/moduleraid/dist/moduleraid.iife.js:1:1001)
    at HTMLScriptElement.<anonymous> (<anonymous>:4:55)
  
> moduleraid.iife.js:1 [moduleRaid] Pushing moduleRaid.arrayArguments[1] into webpackJsonp failed:
TypeError: window[t.entrypoint].push is not a function
TypeError: window[t.entrypoint].push is not a function
    at https://unpkg.com/moduleraid/dist/moduleraid.iife.js:1:1662
    at Array.forEach (<anonymous>)
    at o.fillModules (https://unpkg.com/moduleraid/dist/moduleraid.iife.js:1:1562)
    at new e (https://unpkg.com/moduleraid/dist/moduleraid.iife.js:1:1001)
    at HTMLScriptElement.<anonymous> (<anonymous>:4:55)

Maybe I'm doing something wrong with the configuration? Any suggestion?

**Comment by pixeldesu** [comment]
Unknown Webpack structure means it found an entrypoint, but any of the possible injection methods didn't work. So either it uses a newer one I haven't added yet, or something else is different about it.

Looking at the page, it indeed seems like that is a different `webpackJsonp` method. I'll take a look at it and see if I can add support for it.

**Comment by pixeldesu** [comment]
@marcomow Hey, I just released moduleRaid v6.0.1, a bugfix which should make this work again. Please tell me if the modules are detected now! (for your script, you might need to adjust the unpkg URL to load the actual version instead of the latest package, in case cache loads an older version)

**Comment by marcomow** [comment]
@pixeldesu incredibly quick reaction, thank you!

I can confirm it works also on my end. Only thing left: understanding how to reverse engineer the calculator functions 😅
